snappy/configUpdate.py

```diff
@@ -0,0 +1,9 @@
+# Updates a configuration with new values
+def configUpdate(original, update):
+  for key in update:
+    if (key in original and
+        isinstance(original[key], dict) and
+        isinstance(update[key], dict)):
+      configUpdate(original[key], update[key])
+    else:
+      original[key] = update[key]
```

Comment:
One thing this does is if the two configs have different shapes, the result is whatever the update is. Is that what you want? Would it be better for the `configUpdate` to raise an error when the two dicts have different shapes figuring that indicates something has gone awry?